docs/core/deploying/single-file.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Single file application
+description: Learn what single file application is and why you should consider using this application deployment model.
+author: lakshanf
+ms.author: lakshanf
+ms.date: 08/28/2020
+---
+# Single file deployment and executable
+
+Bundling all application dependent files into a single binary provides an application developer with the attractive option to deploy and distribute the application as a single file. This deployment model has been available since .NET Core 3.0 and has been enhanced in .NET 5.0 where the code is directly executed from the single file in the client machine. On .NET Core 3.0 version, all files were first self-extracted into a temporary location before running. On Windows .NET 5.0, single-file option will not bundle native binaries since a good debugging experience requires these not to be included in the single file currently. Linux will have all files bundled in .NET 5.0.
+
+Single File deployment is available for both [framework-dependent deployment model](index.md#publish-framework-dependent) and [self-contained application](index.md#publish-self-contained) although we anticipate that most developers will use Single file with client-side [self-contained application](index.md#publish-self-contained) managed applications for .NET Core 5.0. The single file deployment option can be combined with [Trim](trim-self-contained.md) and [ReadyToRun](../tools/dotnet-publish.md) publish options.
```

Comment:
I would either omit the "although we anticipate" clause or be more specific about the applicable scenario(s).